index.html

```diff
@@ -128,5 +133,20 @@ <h2 class="color-grey hide flash" id="mobile-message"></h2>
             "logo": "https://zenplayer.audio/img/zen-audio-player-905.png"
         }
     </script>
+    <script>
+    var basic = document.getElementById("themeselector1");
```

Comment:
Let's use jQuery click handlers for consistency, and they can be moved to `js/everything.js`